server/src/utils/asyncHandler.ts

```
@@ -0,0 +1,8 @@
+import { Request, Response, NextFunction } from "express";
+export const asyncHandler = (reqHandler: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)